packages/web-components/fast-components/docs/api-report.md

```diff
@@ -12,9 +12,9 @@ import { BaseProgress } from '@microsoft/fast-foundation';
 import { Breadcrumb } from '@microsoft/fast-foundation';
 import { BreadcrumbItem } from '@microsoft/fast-foundation';
 import { Button } from '@microsoft/fast-foundation';
-import { Card } from '@microsoft/fast-foundation';
 import { Checkbox } from '@microsoft/fast-foundation';
 import { ColorRGBA64 } from '@microsoft/fast-colors';
+import { DesignSystem } from '@microsoft/fast-components-styles-msft';
```

Comment:
Is this right? It seems to be the legacy design system?

Reply:
Looks like it's coming from a previous commit to this file: https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/src/card/index.ts

Reply:
Yeah it’s unrelated here. If we want to fix I’d suggest a PR to address the actual code rather than the API report. If we want to wait on this for that, I’m fine with that - this just needs to go into the release.

Reply:
Yup, just happened to catch it in the api-report, but it should be resolved before we publish, otherwise that change will break us since we dont take a dependency on that package in `@microsoft/fast-components`.

Reply:
@eljefe223 is going to fix this and will re-run the API report.